run-dev.sh

```diff
@@ -9,17 +9,17 @@ CHAINSPEC=$(mktemp -t chainspec_XXXXXXXX --suffix .toml)
 TRUSTED_HASH="${TRUSTED_HASH:-}"
 
 # Generate a genesis timestamp 30 seconds into the future, unless explicity given a different one.
-OFFSET="${OFFSET:-30}"
-TIMESTAMP=${GENESIS_TIMESTAMP:-$(date '+%s000' -d "+$OFFSET sec")}
+TIMESTAMP=$(python3 -c 'from datetime import datetime, timedelta; print((datetime.utcnow() + timedelta(seconds=20)).isoformat("T") + "Z")') 
```

Comment:
Can't `date` do that as well?

Reply:
I don't know really. I copied this from the `nctl` code that Fraser wrote.